### ISSUE TITLE
Mumble client prevents display timeout sleep

### ISSUE THREAD

**Issue by cobrafast** [mumble-voip/mumble#5789]
### Description

After doing the update from 1.3.X to 1.4.230 for Windows 10 (64b), my monitors no longer turn off for power saving while mumble is running. This is especially noticable on the lock screen which has a short display timeout (~1min).

Being connected to a server or not makes no difference.

I found #1394 which was solved by updating audio drivers, however I am on the latest audio drivers (which are dated December 2017) available for my mainboard (ASUS X299 Deluxe). Also, this was not a problem with mumble versions 1.3 or older.

`powercfg -requests` shows no difference between mumble running or not.

### Steps to reproduce

1. Verify energy/display timeout settings
2. Start mumble
3. (Optional) lock screen
4. Wait and observe how the monitor(s) stay on

### Mumble version

1.4.230

### Mumble component

Client

### OS

Windows

### Reproducible?

Yes

**Comment by Krzmbrzl** [comment]
@davidebeatrici could this somehow be related to the new shortcut implementation on Windows? E.g. it permanently generating some sort of input events making the display think it can't turn off?

**Comment by cobrafast** [comment]
> @cobrafast is there an option under Settings > Global shortcuts to disable all shortcuts? If so, could you try doing that to see if the issue persists?

I can't find any such setting. Removing my one shortcut and unchecking *'Enable shortcuts in privileged applications'* had no effect.

**Comment by cobrafast** [comment]
After checking the developer console, I saw that the shortcut system is hooking into my older Logitech joystick (Extreme 3D Pro model). Unplugging it fixed the problem.  
Unchecking *'Enable XInput'* does not help when the joystick is plugged in.

**Comment by Krzmbrzl** [comment]
Good catch! So it somehow _is_ related to the new shortcut implementation.

Iirc XInput is only relevant for Xbox-controllers used under Windows...

**Comment by davidebeatrici** [comment]
> Iirc XInput is only relevant for Xbox-controllers used under Windows...

Correct.

I wonder if the joystick is sending "phantom" events that are registered by Windows (and in turn Mumble).

Here's some test applications: [Joystick-Input-Examples_e2f196877de59134ac1499351d8e7391f4c6a3e0.zip](https://github.com/mumble-voip/mumble/files/9311442/Joystick-Input-Examples_e2f196877de59134ac1499351d8e7391f4c6a3e0.zip)

**Comment by Krzmbrzl** [comment]
@cobrafast could you run the test application `combined.exe` and report its output?
